api/definitions.yaml

```diff
@@ -241,6 +241,28 @@ Service:
   - department_name
   - authorized_cidrs
   - authorized_recipients
+ServicePublic:
+  title: Service (public)
+  description: A Service tied to an user's subscription.
+  type: object
+  properties:
+    service_id:
+      $ref: "#/ServiceId"
+    service_name:
+      $ref: "#/ServiceName"
+    organization_name:
+      $ref: "#/OrganizationName"
+    department_name:
+      $ref: "#/DepartmentName"
+    version:
+      type: integer
```

Comment:
is it possible to define a non-negative integer with swagger constrainta?

Reply:
yes but then we have to put this constraint on all the non negative integer fields which is probably beyond this PR scope

Reply:
agreed